### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by drewvolz**
Information on:
- How to report a vulnerability for this project
- Which versions are currently being supported with security updates
- What to expect from the maintainers once a report is received.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## Supported versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x.x   | :white_check_mark: |
+| < 2.0   | :x:                |
+
+## Reporting a vulnerability
+
+To report a security issue, please email [allaboutolaf@frogpond.tech](mailto:allaboutolaf@frogpond.tech) with a description of the issue, the steps you took to create the issue, affected versions, and, if known, mitigations for the issue. Our team will respond within 3 working days of your email. If the issue is confirmed as a vulnerability, we will open a Security Advisory. This project follows a 90 day disclosure timeline.
```
Comment (rye) [inline SECURITY.md]: To make markdown copy more diffable, I like to ensure that each sentence is on a line. This should still render as one paragraph.

```suggestion
To report a security issue, please email [allaboutolaf@frogpond.tech](mailto:allaboutolaf@frogpond.tech) with a description of the issue, the steps you took to create the issue, affected versions, and, if known, mitigations for the issue.
Our team will respond within 3 working days of your email.
If the issue is confirmed as a vulnerability, we will open a Security Advisory.
This project follows a 90 day disclosure timeline.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+We take security issues in our project seriously and appreciate the efforts of the community in improving the security of our project.
+
+## Supported versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x.x   | :white_check_mark: |
```
Comment (rye) [inline SECURITY.md]: Do we actually support 2.1?

I don't think we're releasing hotfixes — actually, I don't even know if it's possible to release a patch to 2.1.x with the App Store?

Let's just include the most recent version here.

**Review by rye — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (drewvolz) [reply SECURITY.md]: That's a great tip!

File: SECURITY.md (hunk 2 above)
Comment (drewvolz) [reply SECURITY.md]: Good point, adjusted to just note 2.7.

**Review by rye — APPROVED** [review]

**Review by hawkrives — APPROVED** [review]

**Pull request merged**